themes/default/content/registry/packages/vultr/installation-configuration.md

```diff
@@ -23,6 +23,6 @@ Use `pulumi config set vultr:<option>`.
 
 The following configuration points are available for the `vultr` provider:
 
-- `vultr:apiKey` (environment: `VULTR_API_KEY`) -  This is the Vultr API key. It can be found in the [Vultr API section](https://my.vultr.com/settings/#settingsapi).
+- `vultr:apiKey` (environment: `VULTR_API_KEY`) -  This is the Vultr API key. It can be found in the Vultr API section.
```

Comment:
i spent like 20 minutes trying to find where in their docs they say where to get their api keys but i could find nothing and this current link is behind a sign in. so just removing, if anyone knows where the docs are pls let me know.

Reply:
Ok, pls remove it's fine! Idea was to have a quick link for people with a vultr account!

Reply:
@dirien im very happy to include links, but we cant use links that are behind a sign in because they 403

Reply:
Too bad we can't/may not link to such pages. Vultr points to exactly this page from their CLI repo:

https://github.com/vultr/vultr-cli#authentication